### PR TITLE
fix #77098 zonarutoppuden.com

### LINKED ISSUE #77098: zonarutoppuden.com
### Issue URL (Annoyance)
 [https://www.zonarutoppuden.com/2011/04/naruto-shippuden-capitulos.html#](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.zonarutoppuden.com%2F2011%2F04%2Fnaruto-shippuden-capitulos.html%23)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Language-specific:</b><br/> AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by wout-codes**
#77098

### CODE REVIEW

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -37,6 +37,7 @@ giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mtlblog.com,tesisatma
...
 !
 haber365.com.tr##.share-content > a
+zonarutoppuden.com#?#.descrip_title:contains(Compartelo!)
```
[the remark this answers is not in the document]
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:40]: Uggh. lol. I am noot good in CSS

[the remark this answers is not in the document]
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:40]: @zloyden Which one should I add? Please be clear.

[the remark this answers is not in the document]
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:40]: @zloyden And I told you to make the `$$` rule here a generic one. If it was a generic, the reporter may not even have reported it.

[the remark this answers is not in the document]
Comment (zloyden) [reply SocialFilter/sections/specific.txt:40]: `zonarutoppuden.com#?#.descrip_title:contains(Compartelo!)` remove it.
Add rules from this [post](https://github.com/AdguardTeam/AdguardFilters/pull/77102#discussion_r591087625).

[the remark this answers is not in the document]
Comment (zloyden) [reply SocialFilter/sections/specific.txt]: > And I told you to make the `$$` rule here a generic one. If it was a generic, the reporter may not even have reported it.

As far as I remember, you got the answer about generic rules. https://github.com/AdguardTeam/AdguardFilters/pull/76457#discussion_r587282118

**Review by zloyden — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (wout-codes) [reply SocialFilter/sections/specific.txt]: -_-

**Pull request merged**